README.md

```diff
@@ -386,6 +385,54 @@ Private properties should be declared in class extensions (anonymous categories)
 @end
 ```
 
+## Method Ordering
+
+Method implementations are grouped and ordered with lifecycle methods first, then by class inheritance. A `pragma mark -` is used to separate each group.
+
+“Lifecycle methods” are roughly defined, for the purpose of ordering, as those methods dealing with the creation or destruction of instances: initializers, `+alloc`, `-dealloc`, `-initialize`, `-load`, factory methods, `NSCoding` methods, and the like.
```

Comment:
> Its documentation seems to tie it quite tightly to object creation and persistence, ie. lifecycle:

We’re conflating persistence and lifecycle here. Archiving an object does not really imply anything about that object’s life. The object is just as alive as it was before, all that has happened is that it’s current state has been remembered on disk or somewhere else in memory. This is the problem. Lifecycle is effectively meaningless once we start adding concepts like persistence to it. At that point many of the `NSManagedObject` methods, magically become lifecycle methods.